docs/config_to_support_additional_workspace_directories.md

```diff
@@ -0,0 +1,101 @@
+## Config to support additional workspace directories
+
+### Issue
+https://github.com/runfinch/finch/issues/107
+
+### Approach
+
+Lima yaml file has Mounts field to handle the mount points. Similar to other Finch config like cpu and memory, it is straightforward to write additional_workspace_directories to Lima’s override.yaml file.
```

Comment:
[finch.yaml](https://github.com/runfinch/finch/blob/main/finch.yaml#L35) also has config for mounts.

Reply:
Added explanation about why using override.yaml instead of finch.yaml.